help/en/html/doc/Technical/JUnit.shtml

```diff
@@ -880,107 +911,111 @@ setenv JMRI_OPTIONS -Djmri.util.JUnitUtil.checkRemnantThreads=true
         - useful ones in our case could be headless/not headless,
         hardware specific (loco buffer attached, NCE PowerPro
         attached, etc)</li>
 
-        <li><a href=
-        "http://junit.sourceforge.net/javadoc/org/junit/Assume.html">
-        Assumptions</a> - to conditionally ignore a test. For
-        example, a test that would fail in a headless environment
-        can be ignored in headless mode if the first line of the
-        test method is:<br>
-        <code>Assume.assumeFalse(GraphicsEnvironment.isHeadless());</code></li>
-        
+        <li>JUnit 5 provides a collection of <a
+                  href="https://junit.org/junit5/docs/5.1.0/api/org/junit/jupiter/api/condition/package-summary.html">conditional annotations</a>.
+              JMRI makes extensive use of <a href=
+          "https://junit.org/junit5/docs/5.1.0/api/org/junit/jupiter/api/condition/DisabledIfSystemProperty.html">
+              <code>@DisabledIfSystemProperty</code></a> to conditionally ignore a test. For example, a test that would
+              fail in a headless environment should be ignored in headless mode.  Adding the following annotation to
+              the tet method<br>
+          <code>@DisabledIfSystemProperty(named ="java.awt.headless", matches ="true")</code>
+          <br>
+              will cause the test to be ignored in headless mode.  If a whole class is to be disable din headless
+              mode, this same annotation may be applied to the test class.
+      </li>
+
         <li>
         <a href="http://builds.jmri.org/jenkins/job/Development/job/Ignored%20Test%20Scan/lastBuild/testReport/">
                     <img src="http://builds.jmri.org/jenkins/job/Development/job/Ignored%20Test%20Scan/warnings7/trendGraph/png?url=PRIORITY" align="right">
-                </a>        
-        <a href="http://junit.sourceforge.net/javadoc/org/junit/Ignore.html">Ignore</a> - 
+                </a>
+        <a href="https://junit.org/junit5/docs/5.0.0-M2/api/org/junit/jupiter/api/Disabled.html">@Disabled</a> -
         mark a test to be unconditionally ignored. For
         example, a test that fails because it isn't fully implemented yet can be marked to be ignored:<br>
 <pre style="font-family: monospace;">
-        @org.junit.Ignore("not done yet")
+        @org.junit.jupiter.api.Disabled("not done yet")
         @jmri.util.junit.annotations.ToDo("Need to create some mock Framistat Objects")
         @Test
         public void notDoneYet() {
             // some code that compiles but doesn't run
         }
 </pre><br>
-        You should provide the reason for ignoring this test in the <code>@Ignore</code> argument. 
-        <code>@Ignore</code> without an argument
+        You should provide the reason for ignoring this test in the <code>@Disabled</code> argument.
+        <code>@Disabled</code> without an argument
```

Comment:
Would this be a good place to add a discussion of `@ToDo`?  If so, I can do that using your explanation from #8845 after this is merged.

Reply:
There is already a note about todo just after this 

https://github.com/JMRI/JMRI/blob/0b07c55eb5708a21e155e7d4b1c6d16fbc24ea33/help/en/html/doc/Technical/JUnit.shtml#L1004-L1008

There was no need to update the part about ToDo.